account_lock_date_update/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_account_update_lock_date_full,Full access on account.update.lock_date,model_account_update_lock_date,account.group_account_manager,1,1,1,1
```

Comment:
The docs are a bit misleading after this change. A specific group is no longer used to control this access, but instead the general `account_manager` one, and I think it makes more sense, indeed.

However, maybe change `USAGE` to reflect that change.

Reply:
To be fair, the group has not changed. It is still account.account_group_manager. It is called `account adviser` in the Usage because that was the Odoo terminology for this group up until Odoo 10.0. So indeed @alexis-via, perhaps it is time to adapt to 14.0's "Billing Administrator".